course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -225,6 +225,7 @@ def setUp(self):
         super().setUp()
         self.request = make_request()
         self.course = CourseFactory(partner=self.partner)
+        self.course2 = CourseFactory(partner=self.partner, authoring_organizations__key='edx')
```

Comment:
Considering the default course run id on edX `course-v1:edX+DemoX+Demo_Course`
```
self.edx_org_short_name = 'edX'
self.course_2 = CourseFactory(partner=self.partner, authoring_organizations__key=self.edx_org_short_name)
```
and in test cases below use it for validations.

Reply:
Done